src/pages/FriendsInfo.jsx

```diff
@@ -0,0 +1,46 @@
+import React, { useCallback, useEffect, useState } from "react";
+import { getSongs } from "../services";
+import PageHeader from "../components/PageHeader";
+import SongList from "../components/SongList";
+import { connect } from "react-redux";
+import styled, { keyframes } from "styled-components";
+import Spinner from "../components/Spinner";
+
+function FriendsInfo({ username, spotifyId }) {
+  let [songs, setSongs] = useState([]);
+
+  const fetchSongs = useCallback(() => {
+    if (!spotifyId) return;
+    getSongs(spotifyId).then((songs) => {
+      if (songs.length) {
+        console.log(songs);
+        setSongs(songs);
+      }
+    });
+  }, [spotifyId]);
+
+  useEffect(() => {
+    console.log("fetching songs");
+    setSongs([]);
+    fetchSongs();
+  }, [fetchSongs]);
+
+  return (
+    <React.Fragment>
+      <header className="App-container">
+        <PageHeader title={`Look at ${username}'s Top Songs!`} />
+        <p>{username}'s most listened to songs:</p>
```

Comment:
These kinda say the same thing. Maybe we can remove the paragraph or change something else

Reply:
It looks really really empty if we remove either one, maybe we can decide nearer the end to see if we can find copy that works for it, or if we just replace it with something else entirely